xarray/core/concat.py

```diff
@@ -161,6 +161,48 @@ def concat(
     --------
     merge
     auto_combine
+
+    Examples
+    --------
+    >>> da = xr.DataArray(np.arange(6).reshape(2, 3),
+    ...                   [("x", ["a", "b"]),
+    ...                   ("y", [10, 20, 30])])
```

Comment:
I think it would be good to show what the original object looks like. Could you add a line containing `>>> da` here?